### ISSUE TITLE
Strong name signing of NuGet packages

### ISSUE THREAD

**Issue by adamples** [nats-io/nats.net.v1#291]
It seems that 0.6.1 was the last version where NuGet package assemblies were strong name signed. Is this intentional? From #48 I understand that the plan was to still sign official packages:

> NOTE: It should be done automatically when generating the official NUGET package.

**Comment by danielwertheim** [comment]
Thanks for reporting.

@ColinSullivan1 I just downloaded the latest package from NuGet and verified with `sn.exe -v` for both the DLLs, that they are not signed.

In the new solution that has been merged to `master` the pipeline build takes care of this: https://github.com/nats-io/nats.net/blob/master/azure-pipelines.yml#L30

I downloaded an artifact from the new pipeline and verified that `sn.exe -v` says it's valid.

Bump and make a `v0.9.1` release?
